packages/@react-spectrum/s2/src/ActionButton.tsx

```diff
@@ -181,6 +181,7 @@ function ActionButton(props: ActionButtonProps, ref: FocusableRef<HTMLButtonElem
   let domRef = useFocusableRef(ref);
   let overlayTriggerState = useContext(OverlayTriggerStateContext);
 
+  console.log(props.size);
```

Comment:
remove?

Reply:
yep i got it in a commit, just haven't pushed it up yet